packages/opc-feedback/src/opc-feedback.scss

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2021 Red Hat One Platform
+ *
+ * MIT
+ *
+ * opc-feedback.scss consist of style properties for the opc-feedback-component
+ *
+ * @summary opc-feedback.scss will contain the properties for opc-feedback component styles
+ * @author Rigin Oommen
+ *
+ * Created at     : 2021-01-18 14:53:24 
+ * Last modified  : 2021-01-25 20:46:17
+ */
+@import "@one-platform/opc-styles";
+
+$LOCAL: feedback;
+//Add your variables here;
+$variables: ();
+
+:host {
+  --op-transition--default: 120ms ease-in-out;
+  --op-feedback__btn-primary: #b70202;
+  --op-feedback__bg-primary: #fceaeb;
+
+  position: relative;
+  z-index: 9000;
+}
+
+* {
+  outline-width: medium;
+  font-family: "Red Hat Text", "Open Sans", Arial, Helvetica, sans-serif;
+}
+
+.op-feedback {
+  &__panel {
+    position: fixed;
+    bottom: 5.25rem;
+    right: 1.5rem;
+    margin-right: 0;
+    padding: 1.25rem 1.5rem;
+    width: 17rem;
+    max-width: calc(100vw - 3rem);
+    background: white;
+    border: none;
+    border-radius: 0.75rem;
+    border-bottom-right-radius: 0.25rem;
+    box-shadow: 0 0.25rem 0.5rem rgba(0, 0, 0, 0.24);
+    overflow: hidden;
+    transition: all var(--op-transition--default);
+    display: block;
+    opacity: 0;
+    transform: translateY(2rem);
+    perspective-origin: bottom right;
+    &[open] {
+      transform: translateY(0);
+      opacity: 1;
+    }
+
+    &:not([open]) {
+      pointer-events: none;
+    }
+
+    &[open] + .op-feedback__button {
+      background: opc-get-var(BackgroundColor, $opc-color--red-300);
+      box-shadow: 0 0.5rem 0.5rem rgba(0, 0, 0, 0.24);
+    }
+
+    &[open] + .op-feedback__button {
+      border-radius: 2rem;
+      border-top-right-radius: 0.25rem;
+    }
+  }
+  &__button {
+    position: fixed;
+    bottom: 1.5rem;
+    right: 1.5rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background: opc-get-var(BackgroundColor, $opc-color--red-200);
+    color: opc-get-var(TextColor, $opc-color--white);
+    border: none;
+    line-height: 2rem;
+    padding: 0.5rem 1.25rem;
+    font-size: 1rem;
+    cursor: pointer;
+    border-radius: 2rem;
+    border-bottom-right-radius: 0.25rem;
+    box-shadow: 0 0.25rem 0.5rem rgba(0, 0, 0, 0.16);
+    transition: all var(--op-transition--default);
+    &:hover,
+    &:focus {
+      background: opc-get-var(BackgroundColor, $opc-color--red-300);
+      box-shadow: 0 0.5rem 0.5rem rgba(0, 0, 0, 0.24);
+    }
+  }
+  &__subtitle {
+    color: var(--pf-global--Color--200);
+    &.pf-u-text-align-center.pf-u-p-xs {
+      cursor: pointer;
+    }
+  }
+  &__chip {
```

Comment:
why are we using chip style in feedback?

Reply:
https://github.com/1-Platform/op-components/pull/190/files#diff-81cd65b1df3b70deeaf920a405541db28cd4cbffce10888cbdeceb562758a2f3R105

Reply:
In its needed for button chips